### PR TITLE
Feature/fuzz testing

### PR DESCRIPTION
**Pull request by naruto0913**
fuzz testing for adapters and other contracts

### CODE REVIEW

File: v1-contract/hardhat.config.js [hunk 1]
```diff
@@ -13,9 +13,11 @@ module.exports = {
   networks: {
     hardhat: {
       allowUnlimitedContractSize: true,
+      // chainId: 31337,
```
Comment (daiki-iu) [inline v1-contract/hardhat.config.js:16]: @abe-sinjiro  remove this commented code.

File: v1-contract/hardhat.config.js [hunk 2]
```diff
@@ -13,9 +13,11 @@ module.exports = {
...
       forking: {
         url: "https://bsc-dataseed1.binance.org/",
         enabled: true,
+        // url: "https://bscrpc.com",
```
Comment (daiki-iu) [inline v1-contract/hardhat.config.js:20]: @abe-sinjiro  remove this commented code.

**Review by daiki-iu — CHANGES_REQUESTED** [review]

File: v1-contract/hardhat.config.js (hunk 1 above)
Comment (abe-sinjiro) [reply v1-contract/hardhat.config.js:16]: Ok

File: v1-contract/hardhat.config.js (hunk 2 above)
Comment (abe-sinjiro) [reply v1-contract/hardhat.config.js:20]: Ok

File: v1-contract/contracts/HedgepieInvestor.sol [hunk 3]
```diff
@@ -120,6 +120,7 @@ contract HedgepieInvestor is Ownable, ReentrancyGuard {
      * @param _tokenId  YBNft token id
```
[the remark this answers is not in the document]
Comment (abe-sinjiro) [reply v1-contract/contracts/HedgepieInvestor.sol:121]: Will implement and let you know

**Pull request merged**